src/InputField/index.js

```diff
@@ -214,6 +220,7 @@ InputField.defaultProps = {
     focus: false,
     disabled: false,
     required: false,
+    styles: {},
```

Comment:
I am not certain we want to allow this.

Reply:
I think we shouldn't, I'll revert some changes.
It was harder to style the fieldset in combination with the legend element, 
but we def. shouldn't allow overriding styled (otherwise it'll be done),
thanks @varl